cmd/kustomizepatch/command.go

```diff
@@ -0,0 +1,115 @@
+package kustomizepatch
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+)
+
+const (
+	name             = "kustomizepatch"
+	shortDescription = "Generate application configuration as a kustomize patch."
+	longDescription  = `
```

Comment:
I've taken advantage of `longDescription` to dump some knowledge that is dispersed around code in various comments.
I hope summarizing it all in one place will it less frustrating if someone has to figure out all the inputs this command needs.